connectors/ssh/src/main/java/org/crsh/ssh/SSHPlugin.java

```diff
@@ -139,8 +135,8 @@ public void init() {
       File f = new File(serverKeyPath);
       String keyGen = getContext().getProperty(SSH_SERVER_KEYGEN);
       if (keyGen != null && keyGen.equals("true")) {
-        // keyPairProvider = new PEMGeneratorHostKeyProvider(serverKeyPath, "RSA");
-        keyPairProvider = new SimpleGeneratorHostKeyProvider(new File(serverKeyPath));
+        // This seems to provide PEM file generation/loading
```

Comment:
"Seems"? Please either remove the comment or change it to "This provides PEM file generation/loading".